azure-pipelines.yml

```diff
@@ -11,24 +11,57 @@ schedules:
       - dev
   always: true
 
-pool:
-  vmImage: 'ubuntu-latest'
+# =================================================
```

Comment:
should this just go into a new file?

Reply:
I have to admit I wasn't sure how to do that. The name `azure-pipelines.yml` appears to be magic; I'm not sure how to have multiple pipelines defined and have them recognised by the Azure Pipelines infrastructure. I was hoping Beth would show us that.

Reply:
That's set in VSO actually.

Reply:
I'm not surprised :-)